pepper-apis/dsm-core/src/main/resources/liquibase/singular/PEPPER-104_care_of.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="PEPPER-104_care_of" author="pegah">
+        <addColumn tableName="ddp_kit_request_settings">
+            <column name="has_care_of" type="TINYINT(1)"></column>
+        </addColumn>
+    </changeSet>
+</databaseChangeLog>
```

Comment:
if that setting should go out to singular on prod without us waiting for IRB, please also add a update query for ddp_instance_role table

Reply:
It needs IRB Approval unfortunately we have to wait